### PR TITLE
Update for Swift 3.0

### PR DESCRIPTION
**Pull request by keith**
Depends on https://github.com/br1sk/Sonar/pull/5

### CODE REVIEW

File: Brisk/Extensions/Data+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension Data {
+    func toJsonDictionary() -> [String: Any]? {
```
Comment (sberrevoets) [inline Brisk/Extensions/Data+Extension.swift]: JSON*

File: Podfile.lock [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-  Alamofire: 6aa33201d20d069e1598891cf928883ff1888c7a
-  Sonar: 76308d73b94ba4c7e7552c12697ad88c350cb78a
+  Alamofire: 7682d43245de14874acd142ec137b144aa1dd335
+  Sonar: 341e02f0f66c8c4b96128dacebe867cf24c09a98
 
-PODFILE CHECKSUM: bf956e34c1f0d3f2d20fab41231da39a95f96c38
+PODFILE CHECKSUM: 3a052070c76b19c30c3e6dc68054a4203a6b7ff6
 
-COCOAPODS: 1.1.0.beta.2
+COCOAPODS: 1.1.0.rc.2
```
Comment (sberrevoets) [inline Podfile.lock]: Maybe re-run with 1.1 final or 1.2 beta?

**Comment by sberrevoets** [comment]
🚀

File: Podfile.lock (hunk 2 above)
Comment (keith) [inline Podfile.lock]: yea, you guys are too slow 😛

**Pull request merged**